## DenovolyzeR
***

### What is this app?

This app is a simple demonstration of statistical framework implementation DenovolyzeR developed by James Ware of Harvard University and also as a production of [_Autism Sequencing Consortium_](https://genome.emory.edu/ASC/). The purpose of this application is to give the class an idea/hands-on experence of how to analyze de novo variants statistically as well as a touch of user-friendly bioinformatics. 

This app will allow users to conduct 3 types of de novo analysis: assessing the genome wide burden of de novo variants, assessing the numeber of genes with mutiple de novo variants and assessing the frequency of de novo variants in individual genes. User will only have access of preloaded data from an [_autism study dataset_](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4222185/) in this demostration. This demonstration is based on process described by [_this paper_](https://www.ncbi.nlm.nih.gov/pubmed/26439716). 

<br> 


### Who made it?

DenovolyzeR is an R package created by James Ware, as it's official website can be found [_here_](http://denovolyzer.org/).

This denostration is implementated by Yuefu Jiang using [_shinyR_](https://shiny.rstudio.com/). The project code can be found on [_Github_](https://github.com/teruzuki/DenovolyzeR-Demonstration). 

